test/features/parsoid/transform.js

```diff
@@ -111,7 +111,7 @@ parallel('transform api', function() {
         .then(function (res) {
             assert.deepEqual(res.status, 200);
             assert.contentType(res, contentTypes.html);
-            var pattern = /^<h2.*> Heading <\/h2>$/;
+            var pattern = /^<section data-mw-section-id="[^"]+" id="[^"]+"><h2 id="Heading"> Heading <\/h2><\/section>$/;
```

Comment:
@subbuss I've seen your comment regarding this that but it disappeared for some reason (maybe me force-pushing stuff)

> With body only, section tags should be suppressed. we have a mocha test for it.

Can I have a little more background on that? Parsoid should be stripping them right? That means we don't properly pass the `body_only` parameter from RESTBase as I understand...

Reply:
I meant: Parsoid suppresses section tagging in body_only mode because VE requested that. https://github.com/wikimedia/parsoid/commit/e6a6f26141506a93ab5f47c09af0f9ecb49f67d6

Reply:
Oh, ok. This suggests that we have some other bug and `bodyOnly` mode is not passed properly to Parsoid. Looking into that.

Reply:
Hm... Perhaps that fix's not deployed in beta yet? 

`curl -XPOST http://parsoid-beta.wmflabs.org/en.wikipedia.org/v3/transform/wikitext/to/html --data "wikitext=''foo''&body_only=1" ` gives me `<section data-mw-section-id="0" data-parsoid="{}"><p data-parsoid='{"dsr":[0,7,0,0]}'><i data-parsoid='{"dsr":[0,7,2,2]}'>foo</i></p></section>` - so it does have the section wrapping..

Reply:
Ya, just verified - it's not deployed in beta. I will rollback this particular change making a mental note that it's fine that this test fails.

Reply:
ok, let me deploy to beta now.

Reply:
oh, actually, it will require a patch, etc. do you need this now?

Reply:
@subbuss no rush here at all, we can live with a failing test for a few days